docs/content/en/docs/packages/overview.md

```diff
@@ -12,9 +12,17 @@ weight: 1
 Components of EKS Anywhere curated packages consist of a controller, a CLI, and artifacts.
 
 ### Package controller
-The package controller will install, upgrade, configure and remove packages from the cluster. The package controller will watch the packages and packagebundle custom resources for the packages to run and their configuration values. The package controller only runs on the management cluster and manages packages on the management cluster and on the workload clusters.
+The package controller is responsible for installing, upgrading, configuring, and removing packages from the cluster.
 
-Package release information is stored in a package bundle manifest. The package controller will continually monitor and download new package bundles. When a new package bundle is downloaded, it will show up as update available and users can use the CLI to activate the bundle to upgrade the installed packages.
+The package controller performs these actions by watching the package and packagebundle custom resources to determine which packages to run and their appropriate configuration values.
+
+Packages custom resources map to helm charts that the package controller uses to install packages workloads (such as cluster-autoscaler or metrics-server) on your clusters. The packagebundle object is the mapping between the package name and the specific helm chart and images that will be installed.
+
+The package controller only runs on the management cluster and manages packages on the management cluster and on the workload clusters.
+The package controller also runs on single node clusters.
+To learn more about how to install packages on both management and workload clusters, see below.
+
+Package release information is stored in a package bundle manifest. The package controller will continually monitor and download new package bundles. When a new package bundle is downloaded, it will show up as update available and users can use the CLI to activate the bundle to upgrade the installed packages. A package bundle upgrade always requires manual intervention as outlined in the [package bundles docs](./packagebundles.md).
```

Comment:
```suggestion
Package release information is stored in a package bundle manifest. The package controller will continually monitor and download new package bundles. When a new package bundle is downloaded, it will show up as "update available" and users can use the CLI to activate the bundle to upgrade the installed packages. A package bundle upgrade always requires manual intervention as outlined in the [package bundles docs](./packagebundles.md).
```